test/source/mock/lib/oauth.ts

```diff
@@ -60,16 +60,17 @@ export class OauthMock {
     }
   };
 
-  public checkAuthorizationHeaderWithAccessToken = (authorization: string | undefined) => {
+  public checkAuthorizationHeaderWithAccessToken = (authorization: string | undefined, checkAutorization = true) => {
```

Comment:
It looks more like `fallbackAcct` parameter, because even if `checkAuthorization = false`, the lookup happens anyway.
I suggest to refactor the parameter to `fallbackAcct?: string` and pass it from the `google-endpoints.ts` method

Reply:
Sounds good 👍